pkg/kubernetes/scripts/projects.js

```diff
@@ -330,47 +330,59 @@
         'kubeSelect',
         'fields',
         function($q, $scope, projectData, projectPolicy, kselect, fields) {
+            var selectMember = 'Select Member';
+            var NAME_RE = /^[a-z0-9]([-a-z0-9]*[a-z0-9])?$/;
+            var selectRole = 'Select Role';
             var registryRoles = [{ ocRole: "registry-admin", displayRole :"Admin"},
                 { ocRole:"registry-editor", displayRole :"Push" },
                 { ocRole:"registry-viewer", displayRole :"Pull" }];
 
             $scope.select = {
-                member: 'Select Members',
-                members: getAllMembers(),
-                displayRole: 'Select Role',
+                member: selectMember,
+                members: getAllMembers(fields.namespace),
+                displayRole: selectRole,
                 roles: registryRoles,
                 kind: "",
                 ocRole: "",
             };
             
             var namespace = fields.namespace;
-            function getPolicyBinding(namespace){
-                return kselect().kind("PolicyBinding").namespace(namespace).name(":default");
-            }
-            function getAllMembers() {
+
+            function getAllMembers(namespace) {
```

Comment:
Why does this function take a namespace argument? Is it used in the function?

There is also a namespace variable declared right above it, making it confusing. If you really need both I'd suggest renaming one of those variables to make it more readable and avoid potential scope issues.